data/scripts/actions/christmas_bundle.lua

```diff
@@ -0,0 +1,42 @@
+local christmasBundle = Action()
+
+function christmasBundle.onUse(player, item, fromPosition, target, toPosition, isHotkey)
+	local presents = { -- [christmas bundle item id] = {{reward item id, count}, ...}
+		[6507] = { -- red bundle
+			{6569, 15}, {2687, 20}, {2688, 10}, {2675, 10}, {2674, 5}, 6501, 6502, 6490, 6504, 6388
```

Comment:
It would be just as beneficial do the reader to annotate every item here. Perhaps like this?

```lua
{6569, 15}, -- blue fork
{2687, 20}, -- tasty armor
6504, -- a lamp
```

Reply:
yeah indeed, I was thinking the same